azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -53,7 +53,9 @@ def create_from_connection_string(cls, connection_string, ca_cert=None, **kwargs
         :param str connection_string: The connection string for the IoTHub you wish to connect to.
         :param str ca_cert: The trusted certificate chain. Only necessary when using a
             connection string with a GatewayHostName parameter.
-        :param bool websockets: Default is False. Set to true if using MQTT over websockets.
+
+        :param bool websockets: Configuration Option. Default is False. Set to true if using MQTT over websockets.
+        :param str product_info: Configuration Option. Default is empty string. Custom product info to be appended to user agent string.
```

Comment:
>Custom product info to be appended to user agent string. [](start = 80, length = 56)

This sounds like a sentence fragment.  Suggest something like "This string contains arbitrary product info which is appended to user agent string." or something like that.

Reply:
changed the sentence.

---
In reply to: [338260330](https://github.com/Azure/azure-iot-sdk-python/pull/368#discussion_r338260330) [](ancestors = 338260330)